celavi/pylca_celavi/pylca_opt_background.py

```diff
@@ -41,264 +47,11 @@ def model_celavi_lci_background(f_d, yr, fac_id, stage,material, route_id, uslci
        Final LCA results in the form of a dataframe after performing after calculation checks
```

Comment:
Here too, give some idea of the content of the returned DataFrame (emissions or indicators? column names?)

Reply:
I won't leave a comment everywhere, but in general the return values for these functions should have more information

Reply:
I have added detailed information about the returned dataframes for most functions. Some functions are still left. I will keep on adding them. But other than that all other corrections have been completed and the code is running. I will keep an issue open for the final editing for docstrings for me.